### PR TITLE
Result charts

### PR DESCRIPTION
**Pull request by rhantak**
- Add bar charts for user's best and worst ingredients(based on gut feelings) to results page

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,13 @@ script:
   - bundle exec rspec
 services:
   postgresql
+dist: trusty
+addons:
+  chrome: stable
+before_install:
+  - # start your web application and listen on `localhost`
+  - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
+  ⋮
```
Comment (Jonpatt92) [inline .travis.yml]: Nice configuration!

File: app/models/ingredient.rb [hunk 2]
```diff
@@ -8,4 +8,8 @@ class Ingredient < ApplicationRecord
 
   has_many :meal_ingredients, dependent: :destroy
   has_many :meals, through: :meal_ingredients
+
+  def average_gut_feeling(user)
+    meals.where(user_id: user.id).average(:gut_feeling)
```
Comment (Jonpatt92) [inline app/models/ingredient.rb]: Great use of active record

File: .travis.yml [hunk 3]
```diff
@@ -9,6 +9,13 @@ script:
   - bundle exec rspec
 services:
   postgresql
+dist: trusty
+addons:
+  chrome: stable
+before_install:
+  - # start your web application and listen on `localhost`
+  - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
```
Comment (johnktravers) [inline .travis.yml]: Good working fixing the Travis CI issue!

File: app/models/user.rb [hunk 4]
```diff
@@ -19,4 +19,49 @@ class User < ApplicationRecord
...
+    end[0..24]
+  end
+
+  def worst_ingredients
+    ingredients.joins(:meals)
+    .select('ingredients.name, avg(meals.gut_feeling) as avg_gut_feeling')
+    .group('ingredients.id')
+    .having('avg(meals.gut_feeling) < 0')
+    .order('avg_gut_feeling')
+  end
```
Comment (johnktravers) [inline app/models/user.rb]: This ActiveRecord command looks great! Very concise compared to what it was

File: app/views/results/index.html.erb [hunk 5]
```diff
@@ -1,10 +1,124 @@
...
+        return colorArrayBad[1];
+      } else if (info.y <= -2) {
+        return colorArrayBad[2];
+      } else if (info.y <= -1) {
+          return colorArrayBad[3];
+      } else if (info.y <= 0) {
+          return colorArrayBad[4];
+      }
+    }
+  });
```
Comment (johnktravers) [inline app/views/results/index.html.erb]: Good work writing a JavaScript function! The bar graphs look great

File: app/models/user.rb [hunk 6]
```diff
@@ -19,4 +19,49 @@ class User < ApplicationRecord
...
+      }
+    end[0..24]
+  end
+
+  def worst_ingredients
+    ingredients.joins(:meals)
+    .select('ingredients.name, avg(meals.gut_feeling) as avg_gut_feeling')
+    .group('ingredients.id')
+    .having('avg(meals.gut_feeling) < 0')
+    .order('avg_gut_feeling')
```
Comment (Jonpatt92) [inline app/models/user.rb]: Really good.

File: spec/rails_helper.rb [hunk 7]
```diff
@@ -1,73 +1,41 @@
...
+
+Capybara.register_driver :selenium do |app|
+  Capybara::Selenium::Driver.new(app, browser: :chrome)
+end
+
+Capybara.javascript_driver = :selenium_chrome
+
+Capybara.configure do |config|
+  config.default_max_wait_time = 5
+end
```
Comment (Jonpatt92) [inline spec/rails_helper.rb]: Good job figuring out the configuration here.

**Pull request merged**